src/rabbit_memory_monitor.erl

```diff
@@ -86,7 +86,9 @@ report_ram_duration(Pid, QueueDuration) ->
 stop() ->
     gen_server2:cast(?SERVER, stop).
 
-conserve_resources(Pid, disk, Conserve) ->
+%% Paging should be enabled/disabled only in response to disk resource alarms
```

Comment:
It's not just paging that could use the disk. However, RabbitMQ probably tries to be too smart and too conservative at the moment: disk limit is never set to 0 in practice, so writing to disk is not necessarily going to make a node run out of space; we also cannot completely avoid running out of space because it can be another process that's been filling up the disk. The new behaviour is a reasonable compromise between not blocking every other node and not doing anything to avert the problem.